lib/ABI/FunctionType/Layout.cpp

```diff
@@ -813,14 +823,22 @@ Layout::Layout(const model::RawFunctionType &Function) {
     revng_assert(StackArgType.Qualifiers().empty());
     revng_assert(StackArgType.is(model::TypeKind::StructType));
 
+    const model::Type *OriginalStackType = StackArgType.UnqualifiedType().get();
+
     auto &Argument = Arguments.emplace_back();
 
+    // (WIP): I have FUD about this, so I'm leaving it out for now, let's
+    //        discuss RFT's stack argument comment during the review process.
+    //
+    // Argument.Comment = StackArgType.UnqualifiedType().get()->Comment();
```

Comment:
Why not?

Reply:
As per mattermost discussion, we decided to
1. decouple this from the layout,
2. concatenate **type** comments in a fashion similar to
      /// \param stack_args: field_at_offset_0: <comment here>
      ///                    field_at_offset_64: <another comment here>
      /// . . .
3. do the same for RFT return values as well

Reply:
All three are done :tada: